cypress/e2e/pages/viewApplication.cy.ts

```diff
@@ -69,4 +69,75 @@ describe('View a resident application', () => {
     ViewApplicationPage.getErrorSummary().should('be.visible');
     cy.contains(`Unable to delete household member`);
   });
+
+  it('shows the notes and history section for read only users', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application);
+
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection().should('exist', {
+      timeout: 5000,
+    });
+  });
+
+  it('allows readonly users to see note section', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId, true);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application, true);
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection()
+      .click()
+      .get('[data-testid="add-text-input"]')
+      .should('exist');
+  });
+
+  it('Shows overall history but does not show detailed history for readonly users', () => {
+    const application = generateApplication(applicationId, personId);
+    //ensure application requires assessment
+    application.status = ApplicationStatus.SUBMITTED;
+
+    cy.loginAsUser('readOnly');
+    cy.mockActivityHistoryApiEmptyResponse(applicationId, true);
+    cy.mockHousingRegisterApiGetApplications(applicationId, application, true);
+
+    ViewApplicationPage.visit(applicationId);
+    ViewApplicationPage.getNotesAndHistorySection()
+      .click()
+      .get('[data-testid="activity-history"]')
+      .should('exist');
```

Comment:
I guess same as above for any of these - no need to click or get if you're just checking if it exits.